src/pages/experimental/create-flows/index.mdx

```diff
@@ -289,3 +302,45 @@ A notification banner can be presented to visually confirm when any create actio
 
 </Column>
 </Row>
+
+## Pre-create
+
+Pre-create occurs before the normal create flow starts. A common pre-create is _create from template_, however there are many other uses.
+
+Pre-create is usually combined with a full-page create flow. It can also be used in combination with a side panel create flow. Pre-create can be used in combination with single step or multistep create flows. A multistep flow starts **after** pre-create is complete.
```

Comment:
```suggestion
A pre-create step is usually combined with a [full-page create flow](#full-page). It can also be used in combination with a [side panel create flow](#full-page). A pre-create step can be used in combination with a single-step or multistep create flow. A multistep flow starts after the pre-create step is complete.
```